api/app/services/escalation/escalation_repository.py

```
@@ -266,6 +287,27 @@ async def update_rating(self, message_id: str, rating: int) -> bool:
+        try:
+            async with aiosqlite.connect(self.db_path) as db:
+                await db.execute(
+                    """
+                    INSERT INTO escalation_rating_consumed_tokens (
+                        token_jti, message_id, consumed_at
+                    ) VALUES (?, ?, ?)
+                    """,
+                    (token_jti, message_id, now),
+                )
+                await db.commit()
+            return True
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)